data-semantics.md

```diff
@@ -27,7 +27,7 @@ Secondarily, an optional **component** tag can be provided to scope the operatio
 
 ### Span Structure
 
-Span structure is also important: what do spans represent, and what relationship do they have towards their parent span?  This is covered in the [Semantic Specification](/spec).
+Span structure is also important: what do spans represent, and what relationship do they have towards their ancestor spans?  This is covered in the [Semantic Specification](/spec).
```

Comment:
supernit: can replace "ancestor spans" w/ "ancestors" since the word span is already mentioned